llama_index/chat_engine/condense_question.py

```diff
@@ -133,20 +148,23 @@ def chat(
             print(log_str)
 
         # Query with standalone question
-        response = self._query_engine.query(condensed_question)
+        query_response = self._query_engine.query(condensed_question)
+        tool_output = self._get_tool_output_from_response(
```

Comment:
if the query_response has source_nodes, this gets lost right?

Reply:
nope, it ends up in the `raw_output` of the tool output -> see `self._get_tool_output_from_response()`

Reply:
i see